doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -22,10 +22,10 @@ Installing OpenCV-Python from Pre-built Binaries
 
 This method serves best when using just for programming and developing OpenCV applications.
 
-Install package [python-opencv](https://packages.ubuntu.com/trusty/python-opencv) with following command in terminal (as root user).
+Install package [python-opencv](https://packages.ubuntu.com/focal/python3-opencv) with following command in terminal (as root user).
```

Comment:
> `[python-opencv]`

old name is still used.

Reply:
First time to contribute😅, I feel like a fool, thank you for your patience @alalek .